ledger/src/lib.rs

```diff
@@ -92,6 +92,29 @@ pub enum RecordsFilter<N: Network> {
     SlowUnspent(PrivateKey<N>),
 }
 
+/// Helper struct to wrap transmissions in the Subdag.
+pub struct SubdagTransmissions<N: Network> {
+    pub transmissions: IndexMap<TransmissionID<N>, Transmission<N>>,
```

Comment:
are `prior_included_transmissions` and `aborted_transmissions` a part of the `transmissions`?

Reply:
Nope, hope these new comments clarify: 4fc3258f5
Adding an intermediary check that transmissions does not contain aborted feels not worth the compute overhead, and checks down the line should independently verify the validity of the received contents.